components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -1091,7 +1102,7 @@ public Response applicationsApplicationIdMapKeysPost(String applicationId, Appli
                 jsonParamObj.put(APIConstants.JSON_CLIENT_SECRET, body.getConsumerSecret());
                 Map<String, Object> keyDetails = apiConsumer
                         .mapExistingOAuthClient(jsonParamObj.toJSONString(), username, clientId,
-                                application.getName(), keyType, tokenType, keyManagerName, xWSO2Tenant);
+                                application.getName(), keyType, tokenType, keyManagerName, MultitenantConstants.SUPER_TENANT_DOMAIN_NAME);
```

Comment:
We need to pass organization instead of hard coded value right

Reply:
Done in https://github.com/wso2/carbon-apimgt/pull/10498/commits/919bffeafeed49d2dbf9e95596a220a8e9dd2e0b#diff-12f710906e8ac379963b9527b560b589024a79f07c7395c425241e517048a0b0R1106